### ISSUE TITLE
Computer tab layout is not optimal with long location names

### ISSUE THREAD

**Issue by AndiMb** [glpi-project/glpi#14166]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.6

### Bug description

When having long location name or a complex location hierarchie like "Country (plus some text) > State (plus some text) > City (plus some text) > Building (plus some text) > Floor (plus some text) > Room (plus some text)" and a computer is addigned to a location, the layout of the computer tab is not optimal and get larger than the webpage. Also if images are added for the computer model the textfields are hidden by the model images.

without location
![image](https://user-images.githubusercontent.com/5680717/221122373-dcdc3406-8608-4284-8f4c-d1079d3e65ff.png)

with location
![image](https://user-images.githubusercontent.com/5680717/221122463-370f0a40-b84b-4dc5-8338-665ce6d2c5ca.png)

without location and with computer model images
![image](https://user-images.githubusercontent.com/5680717/221124226-f88899a3-c3ff-45c9-bd3c-dc9af85215bb.png)

with location and computer model images
![image](https://user-images.githubusercontent.com/5680717/221124091-186b20f9-fe51-44f5-8bdd-caba18a8380a.png)

### Steps To reproduce

1. Create a location hierarchie with many levels and/or long location names
2. assign the location a computer
3. assign a copmuter model with front and rear images
4. open the computer tab

**Comment by cedric-anne** [comment]
Hi,

I cannot reproduce your issue on GLPI 10.0.7

![image](https://user-images.githubusercontent.com/33253653/231280553-8d8499d9-02c4-4b24-8fe0-eb96b51b8825.png)
![image](https://user-images.githubusercontent.com/33253653/231281742-2a66e4c0-d84c-45da-8fe9-4417e03013a5.png)

Can you try to reproduce? If issue persists, please give some information about the used browser (what browser, what version, what OS, ...).

**Comment by AndiMb** [comment]
Yes, I can reproduce it with GLPI 10.0.7. I'm using Windows 10 22H2 with Edge (112.0.1722.39 64-Bit) and Firefox (112.0 64-Bit). It seems more dependent on the aspect ratio of the browsers than the browsers itself. When changing the browser aspect ratios there are states where the described problems happens.

**Comment by cedric-anne** [comment]
The only poblem that I am able to reproduce is an overflow issue on really small screen:
![image](https://user-images.githubusercontent.com/33253653/231386792-af82e070-cd2a-4eed-a5e3-47c6f87168ce.png)

But I tested on Chrome 112.0 / Firefox 112.0 on Ubuntu.

**Comment by Lainow** [comment]
> Perhaps it also depends on the images. I attached the images.
> 
> As name of the location I used (so the complete name and the name are): "Country (plus some text) > State (plus some text) > City (plus some text) > Building (plus some text) > Floor (plus some text) > Room (plus some text)" I ignored the childlevels and just copied the full string as name of the location.
> 
> ![ComputerImage_Front](https://user-images.githubusercontent.com/5680717/231409270-7a05b7c8-525b-4c73-8611-86fc2223649c.png) ![ComputerImage_Back](https://user-images.githubusercontent.com/5680717/231409276-17239615-051d-4b35-bd5c-119fbdbf5f09.png)

Hello,

I tried to reproduce your bug in the current version of GLPI and it turns out that it has already been solved. Have you tried updating your GLPI?

![image](https://github.com/glpi-project/glpi/assets/107540223/e0425cba-1a50-47a2-a742-6d42bf787003)

**Comment by AndiMb** [comment]
This still happens for us on GLPI 10.0.12.

**Comment by cedric-anne** [comment]
Fixed by #20478.
